### ISSUE TITLE
Webcam Source does not find the camera on Hololens 2.

### ISSUE THREAD

**Issue by SimGu96** [microsoft/MixedReality-WebRTC#475]
**Describe the bug**
If I have the webcam source component as the video track source and running on Hololens 2, I get errors like:
'The stream number provided was invalid.'
'No capture devices are available.'. 

[Output-Debug.txt](https://github.com/microsoft/MixedReality-WebRTC/files/4960005/Output-Debug.txt)

However if I print the first device from `PeerConnection.GetVideoCaptureDevicesAsync()` rigth before WebcamSource.cs is setting the deviceConfig  "`var deviceConfig = new LocalVideoDeviceInitConfig`"
It prints:
" \\?\DISPLAY#QCOM_AVStream_850#3&809a381&0&UID32768#{e5323777-f976-4f5b-9b55-b94699c46e44}\{5584F823-3830-4CFD-947F-78DE17A8B14C} "

Does that mean it can find the camera but not use it for some reason? 

If I use SceneVideoSource on my desktop in Unity insted of WebcamSource on Hololens 2 it works. So the webrtc signaling part should be working.

**Expected behavior**
A video stream sent from my Hololens 2.

**Environment**
- Git branch: master
 - Platform: UWP
 - Architecture:  ARM
 - Unity version (if applicable): [e.g. 2018.4, 2019.3.15f1]
 - Target device:  HoloLens 2

**Comment by djee-ms** [comment]
The camera was detected, but is not available. There is likely another application using it, possibly another instance of the app still runnning; check with the Developer Portal maybe? I tested the master branch yesterday and everything works fine.

**Comment by kiokju** [comment]
I have same issue. My develop Environment is Unity, hololens2, ARM.

'The stream number provided was invalid.'
'No capture devices are available.'.

Same error.

I check Developer Portal. But Not found any camera use App.

**Comment by Mathieu-Allaert** [comment]
@kiokju did you find a solution?

**Comment by kiokju** [comment]
> @kiokju did you find a solution?

Yes. Maybe It was Unity version issue.  After 2019.4.14. This issue is fixed.
